aerogear/data-sync-ui
# Pino v4 -> v5

## Motivation
https://issues.jboss.org/browse/AEROGEAR-7790
## What
* Update of pino logger to v5
* prettify logging in dev mode

## Verification Steps
There's not much anything to verify. If you want, try to install and run the server + create some resource in UI and check that logs look pretty as heaven
1. ```
   rm -rf node_modules && npm i
   docker-compose up &
   npm run server
   ```
2. Go to `localhost:8080` and create some resource, e.g. add data source, schema, resolver etc.
3. Check the logs and be amazed
